src/spectrecoff/Panel.fs

```diff
@@ -14,8 +14,8 @@ let defaultPanelLayout: PanelLayout =
        Sizing = Collapse
        Padding = AllEqual 2 }
 
-let customPanel (layout: PanelLayout) (header: string) (content: OutputPayload) =
-    let panel = Panel(content |> toMarkedUpString)
+let customPanel (layout: PanelLayout) (header: string) (content: string) =
```

Comment:
Funny, is this a coincidence or did you change this because it came up in the ProgressChart PR?

Reply:
Yes, I was inspired by your PR, I must admit.

Reply:
But in my latest push I changed it again :P